index.rst

```diff
@@ -4,22 +4,215 @@
+The type of documentation that is most appropriate to use is dependent upon the situation.
+This document describes the different documentation options and when they may be applicable to your specific case.
+
+.. Important::
+   
+   Questions that are not answered by this document can be asked in the ``#rubinobs-sitcom-docs`` Slack channel. 
+
+
+Project Level Change Controlled Documentation
+=============================================
+
+SIT-Com follows the project standard in that all documentation requiring project-level change control requires an LSE document handle and all modifications to the document are handled through the project Change Control Board (CCB).
+Specifically, this should includes procedures which may result in damage to equipment and/or injury to personnel (e.g. mirror handling) as well as formal ICDs.
+It is anticipated that very few SIT-Com generated artifacts will fall into this category.
+
+SIT-Com Level Change Control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of documentation that is applicable to only SIT-Com personnel and is internal to the SIT-Com subsystem(s), the implementation of a SIT-Com change board has been discussed but is not currently in place.
+Examples of documents that are SIT-Com specific but should be under change control include the documentation of workflows (like this one), management plans, and/or SIT-Com specific procedures that require sign-off by specific domain experts.
+Until such a system is put in place, these types of documentation exist as `Technotes`_.
+
+When required to be under stricter change controls, branch protection should be put in place.
+The designation of specific reviewers can also be manually specified.
+In the event that this requires higher-level SIT-Com repo permissions, please consult the ``#rubinobs-sitcom-docs`` Slack channel for support.
+
+General Documentation Types
+===========================
+
+This section outlines the different types of documentation used by SIT-Com personel.
+
+Jira Tickets
+^^^^^^^^^^^^
+
+SIT-Com related work should follow the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_.
+Documentation in Jira tickets should be sufficiently detailed such that it addresses the description in the task.
+It is not expected to be consulted in the future, especially on a regular occurrence. 
+In many cases, specifically when characterization activities have taken place over a series of tickets, external documentation should be developed and linked to the ticket(s).
+The type of document to be created is case dependent, but will most likely be covered by `Technotes`_ and/or inclusion of a section in the `Obs-Ops Webspace`_.
+The creation of the documentation can be performed under one of the already opened tickets, or, a new ticket can be created that explicitly deals with the generation of documentation.
```

Comment:
"tickets, or, a new" -> "tickets or, a new" (?)

Reply:
apparently we were both wrong...
"tickets, or a new..."